src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -139,7 +146,24 @@ export default class ArcanaTracking extends Module {
 
 		// if they drew 2-3 cards in the first 30 sec, we can assume they had a sleeve draw
 		if (prepullSleeve && _.inRange(sleeveDrawLog.length, SleeveType.TWO_STACK, SleeveType.TWO_STACK + 2)) {
-			this.cardStateLog[0].sleeveState = sleeveDrawLog.length - 1
+			this.cardStateLog[0].sleeveState = sleeveDrawLog.length
+			// prefab a sleeve draw if there isn't already
+			if (!prepullSleeveFabbed) {
+				events.splice(0, 0, {
+					ability: {
+						abilityIcon: sleeveDrawLog.length > 2 ? '019000/019562.png' : '019000/019561.png',
```

Comment:
Stylistic nit more than anything – I'd put these in a variable so it's easier to read (I put typos in paths that are only numbers)

Reply:
Probably should have, it's almost the same as having a magic number